pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BaseBrokerStarter.java

```diff
@@ -256,7 +257,7 @@ public void start()
     // Create Broker request handler.
     String brokerRequestHandlerType =
         _brokerConf.getProperty(Broker.BROKER_REQUEST_HANDLER_TYPE, Broker.DEFAULT_BROKER_REQUEST_HANDLER_TYPE);
-    BrokerRequestHandler singleStageBrokerRequestHandler = null;
+    BaseBrokerRequestHandler singleStageBrokerRequestHandler = null;
```

Comment:
The declaration of multistage handler at line 277 can also be done similarly `BaseBrokerRequestHandler multiStageBrokerRequestHandler` for consistency.?